src/plone/api/tests/test_content.py

```diff
@@ -376,6 +376,64 @@ def test_create_at_with_title_in_request(self):
 
         self.assertEqual(page.title, 'Test document')
 
+    @unittest.skipIf(HAS_PACONTENTYPES, 'Archetypes only')
+    def test_create_at_collection(self):
+        """Test create at Collecition."""
+        collection = api.content.create(
+            container=self.portal,
+            type='Collection',
+            title='Mandelbrot set',
+            description='Image gallery of a zoom sequence',
+            query=[
+                {
+                 'i': 'Type',
+                 'o': 'plone.app.querystring.operation.string.is',
+                 'v': ['Image'],
+                },
+            ],
+        )
+        self.assertEqual(collection.Title(), 'Mandelbrot set')
+
+    @unittest.skipIf(HAS_PACONTENTYPES, 'Archetypes only')
+    def test_create_at_event(self):
+        """https://github.com/plone/plone.api/issues/364"""
+        from DateTime import DateTime
+        today = DateTime()
+        tomorrow = today + 1
+        event = api.content.create(
+            container=self.portal,
+            type='Event',
+            title=u'My event',
+            startDate=today,
+            endDate=tomorrow,
+        )
+        self.assertEqual(event.startDate, today)
+        self.assertEqual(event.endDate, tomorrow)
+        results = api.content.find(Title=u'My event')
+        self.assertEqual(len(results), 1)
+        self.assertEqual(results[0].start, today)
+        self.assertEqual(results[0].end, tomorrow)
+
+    @unittest.skipUnless(HAS_PACONTENTYPES, 'Dexterity only')
+    def test_create_dx_event(self):
+        """https://github.com/plone/plone.app.contenttypes/issues/465"""
```

Comment:
@hvelarde what's the relationship between plone/plone.app.contenttypes#465 and ```plone.api```? It seems to me more a problem of ```plone.app.contenttypes```.

I think this test can continue but without reference to ```plone.app.contenttypes```.

Reply:
IIRC, I was unable to reproduce it on a clean Plone installation; so maybe it's not related at all.

Reply:
I fixed the typos and run test for AT and DX and Travis is green.

I kept the test regarding the event creation, do not know if it is necessary now that we have a test with Collection creation for AT and DX?

Reply:
@gbastien this event testing is necessary to avoid regression #364. I think it is necessary now only to remove the reference:

https://github.com/plone/plone.app.contenttypes/issues/465

Reply:
In fact what avoids #364 is the AT test. But it's good to have an event creation test with DX

Reply:
OK so I would remove the test_create_at_event, unnecessary now because of test_create_collection?

Reply:
@gbastien no. test_create_at_event avoid regression #364. test_create_collection not avoid that.

Reply:
Right, so let's merge? :-)

Reply:
@gbastien I think it is necessary now only to remove the reference:

plone/plone.app.contenttypes#465

in docstring.